src/main/java/uk/ac/ebi/pride/spectracluster/hadoop/merge/MergeClusterByIDReducer.java

```diff
@@ -0,0 +1,79 @@
+package uk.ac.ebi.pride.spectracluster.hadoop.merge;
+
+import org.apache.hadoop.io.Text;
+import uk.ac.ebi.pride.spectracluster.cluster.ICluster;
+import uk.ac.ebi.pride.spectracluster.hadoop.util.AbstractClusterReducer;
+import uk.ac.ebi.pride.spectracluster.hadoop.util.ConfigurableProperties;
+import uk.ac.ebi.pride.spectracluster.hadoop.util.IOUtilities;
+
+import java.io.IOException;
+import java.util.Collection;
+
+/**
+ * MergeClusterByIDReducer clusters using ClusteringEngine
```

Comment:
I'm not quite sure I understand the purpose of this job completely. It basically clusters all clusters that share the same id in one job? Why would clusters share the same id? Is this done just to make sure no duplicate ids are present? But I'm 99% certain it's me not getting something here!!!